static/cdn/maintenance.html

```diff
@@ -1,208 +1,329 @@
+				<div class="content">
+					<h1>The Scratch Team is working hard to fix an issue with the Scratch website</h1>
+					<h2 class="reload" onclick="location.reload()">Try reloading the page in a few minutes</h2>
+					<h2 class="comment">Check <a href="https://twitter.com/scratch">@Scratch</a> for updates</h2>
+					<hr>
+					<div class="narrow">
+						<p>While the website is being fixed, you can use the Scratch app to make projects offline</p>
+					</div>
+					<button class="toggleShowing showing"><span>Download Scratch app</span></button>
+					<div class="grid-wrapper hidden">
+					  <div class="grid-content">
+					    <div class="grid-box">
+								<div class="os-name-row">
+									<div class="os-name-img">
+										<img src="http://scratch-maintenance.s3.amazonaws.com/windows-logo.svg" width="28px" />
+									</div>
+									<div class="os-name-text">Windows</div>
+								</div>
+								<div class="os-version">Windows 10+</div>
+								<div class="download-link-row">
+									<div class="store-img">
+										<a href="https://www.microsoft.com/store/apps/9pfgj25jl6x3?cid=storebadge&ocid=badge">
+											<img src="http://scratch-maintenance.s3.amazonaws.com/windows-store-badge.svg" />
+										</a>
+									</div>
+									<div class="vertical-separator-left"></div>
+									<div class="vertical-separator-right"></div>
+									<div class="download-link"><a href="https://downloads.scratch.mit.edu/desktop/Scratch%20Setup.exe">Download</a></div>
+								</div>
+							</div>
+					    <div class="grid-box">
+								<div class="os-name-row">
+									<div class="os-name-img">
+										<img src="http://scratch-maintenance.s3.amazonaws.com/apple-logo.svg" width="28px" />
+									</div>
+									<div class="os-name-text">macOS</div>
+								</div>
+								<div class="os-version">macOS 10.13+</div>
+								<div class="download-link-row">
+									<div class="store-img">
+										<a href="https://apps.apple.com/us/app/scratch-desktop/id1446785996?mt=12">
+											<img src="http://scratch-maintenance.s3.amazonaws.com/mac-store-badge.svg" />
+										</a>
+									</div>
+									<div class="vertical-separator-left"></div>
+									<div class="vertical-separator-right"></div>
+									<div class="download-link"><a href="https://downloads.scratch.mit.edu/desktop/Scratch.dmg">Download</a></div>
+								</div>
+							</div>
+					    <div class="grid-box">
+								<div class="os-name-row">
+									<div class="os-name-img">
+										<img src="http://scratch-maintenance.s3.amazonaws.com/chromeos-logo.svg" width="26px" />
+									</div>
+									<div class="os-name-text">ChromeOS</div>
+								</div>
+								<div class="os-version">ChromeOS</div>
+								<div class="download-link-row">
+									<div class="store-img">
+										<a href="https://play.google.com/store/apps/details?id=org.scratch">
+											<img src="http://scratch-maintenance.s3.amazonaws.com/google-play-badge.png" />
```

Comment:
I'm not seeing the Google Play Store badges on the preview. Maybe it is out-of-date?

Reply:
Yes, I just updated it, sorry